docs/content/toc.md

```diff
@@ -2,103 +2,103 @@
 ---
 
 ## Getting Started
-  * [Concepts](../design/)
-  * [Quickstart](../tutorials/quickstart.html)
-  * [Loading Data](../tutorials/ingestion.html)
-    * [Loading from Files](../tutorials/tutorial-batch.html)
-    * [Loading from Streams](../tutorials/tutorial-streams.html)
-    * [Loading from Kafka](../tutorials/tutorial-kafka.html)
-  * [Clustering](../tutorials/cluster.html)
+  * [Concepts](/docs/#{DRUIDVERSION}/design/)
```

Comment:
@xvrl's method is probably a bit better as it preserves "latest" when it is the base.